controller/base-controller.js

```diff
@@ -162,6 +164,25 @@ module.exports = class BaseController extends EventEmitter {
     return validator(key, req.form.values[key], req.form.values, emptyValue);
   }
 
+  _sanitize(req, res, callback) {
+    // If we don't have any data, no need to progress
+    if(req.form.values) {
+      Object.keys(req.form.values).forEach(function (property, propertyIndex) {
+        // If it's not a string, don't sanitise it
+        if(_.isString(req.form.values[property])) {
```

Comment:
might be good to have some unit tests to check for values passed to _sanitize in the req.form.values that aren't strings so if this was removed those tests would break

Reply:
Good shout, done.